modules/client-configuration/pages/clients-centos.adoc

```diff
@@ -45,7 +40,6 @@ For example, when working with `x86_64` architecture, you need this products:
 | OS Version | Product Name
 | {centos} 6 | CentOS 6 x86_64
 | {centos} 7 | CentOS 7 x86_64
-| {centos} 8 | CentOS 8 x86_64
```

Comment:
The title say: "Removed CentOS 6 ..." but you remove CentOS **8**

Reply:
ok, I changed the title.